doc/users/github_stats.rst

```diff
@@ -3,17 +3,23 @@
 Github Stats
 ============
 
-GitHub stats for 2014/08/26 - 2016/01/09 (tag: v1.4.0)
+GitHub stats for 2015/02/16 - 2016/07/02 (tag: v1.4.3)
 
 These lists are automatically generated, and may be incomplete or contain duplicates.
 
-We closed 744 issues and merged 877 pull requests.
+We closed 807 issues and merged 894 pull requests.
 
-The following 260 authors contributed 5401 commits.
+The following 316 authors contributed 5340 commits.
 
 * Acanthostega
+* Adrien F Vincent
+* Adrien F. VINCENT
+* Adrien VINCENT
```

Comment:
I don't know how the list is generated, but if you're planning to alter it by hand to remove the duplicated entrie then “Adrien F. Vincent” is fine instead of the 4 duplicated entries due.